bundles/org.openhab.binding.playstation/README.md

```diff
@@ -0,0 +1,117 @@
+
+## Supported Things
+
+This binding should support all PS4 variants.
+It can also tell if your PS3 is ON or OFF/not present.
+
+## Discovery
+
+Discovery should find all your PS4s within a few seconds as long as they are in standby mode and not completely turned off.
+To be able to discover your PS3 you need to turn on "Connect PS Vita System Using Network" in
+Settings -> System Settings -> Connect PS Vita System Using Network.
+
+## Thing Configuration
+
+If you want to control your PS4 the first thing you need is your user-credentials, this is a 64 characters HEX string that is easiest obtained by using PS4-waker https://github.com/dhleong/ps4-waker.
+The result file is called ".ps4-wake.credentials.json" in your home directory.
+
+Then you need to pair your OpenHAB device with the PS4.
+This can be done by saving the Thing while the pairing screen is open on the PS4. The code is only needed during pairing.
+
+Then, if you have a pass code when you log in to your PS4 you have to specify that as well.
+ 
+
+## Channels
+
+| Channel Type ID | Item Type | Description                                                              | Read/Write |
+|-----------------|-----------|--------------------------------------------------------------------------|------------|
+| power           | Switch    | Shows if PlayStation is ON or in standby.                                | RW         |
+| applicationName | String    | Name of the currently running application.                               | R          |
+| applicationId   | String    | Id of the currently running application.                                 | RW         |
+| applicationImage| Image     | Application artwork.                                                     | R          |
+| oskText         | String    | The text from the OnScreenKeyboard.                                      | RW         |
+| disconnect      | Switch    | Disconnect from PS4.                                                     | W          |
+| keyUp           | Switch    | Push Up button.                                                          | W          |
+| keyDown         | Switch    | Push Down button.                                                        | W          |
+| keyRight        | Switch    | Push Right button.                                                       | W          |
+| keyLeft         | Switch    | Push Left button.                                                        | W          |
+| keyEnter        | String    | Push Enter button.                                                       | W          |
+| keyBack         | Switch    | Push Back button.                                                        | W          |
+| keyOption       | Switch    | Push Option button.                                                      | W          |
+| keyPS           | Switch    | Push PS button.                                                          | W          |
```

Comment:
It feels like Actions are way too advanced to do something so simple as to only send a message. Are triggers only for channels that emit an event? But if that is the way they should be I will fix it but it will take some more time then a weekend.

Reply:
You could also consolidate these Channels into one Channel with a Command Description. This should be way easier as migrating to Actions. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description

Reply:
Thanks, will definitely try that instead.